texar/torch/data/tokenizers/pretrained_roberta_tokenizer.py

```diff
@@ -39,7 +39,8 @@ class RoBERTaTokenizer(GPT2Tokenizer):
             If None, the model name in :attr:`hparams` is used.
         cache_dir (optional): the path to a folder in which the
             pre-trained models will be cached. If `None` (default),
-            a default directory (user's home directory) will be used.
+            a default directory (``texar_pytorch`` folder under user's home
```

Comment:
I was told NLTK also put downloaded stuff under home directory. What folder name do they use? Can we do a quick test?

I'm concerning the name `texar_pytorch` can be confusing (e.g., with the repo name `texar-pytorch`). Also, the downloaded stuff is supposed to be shared with texar-tf. Perhaps a name like `texar_downloads` is good (and Texar-TF should use the same folder). But let's take a look at other tools and see what's the convention here.